src/libServer/GetWorkServer.cpp

```diff
@@ -113,26 +113,11 @@ int GetWorkServer::GetSecondsToNextPoW() {
 }
 
 // GetResult returns the Pow Result
-// if wait_ms < 0: wait until first accept result
-// if wait_ms = 0: return current result immediately
-// if wait_ms > 0: wait until timeout, return the last result
-ethash_mining_result_t GetWorkServer::GetResult(const int& wait_ms) {
-  if (wait_ms >= 0) {
-    std::this_thread::sleep_for(std::chrono::milliseconds(wait_ms));
-    return m_curResult;
-  }
-
-  // wait_ms < 0
-  // wait until the first accept result
-  if (!m_isMining || m_curResult.success) {
```

Comment:
should we keep this for these 2 conditions?
1. the pow was already done before call GetResult
2. the pow was stopped before call GetResult

Reply:
I think only need to check m_isMining, because if m_curResult.success is true, the conditional variable must be triggered, then it can return. I don't check m_curResult here because it is not protected by mutex

Reply:
Ok, can add this check